### ISSUE TITLE
Maybe you will be interested - libmdbx

### ISSUE THREAD

**Issue by 0x4E69676874466F78** [zenhotels/lmdb-go#2]
https://github.com/leo-yuriev/libmdbx

**Comment by xlab** [comment]
Yep, seen his talk on HL++ 2015. Although I'd stick to vanilla MDB, until encounter any of the problems that libmdbx addresses. There are too many quirks exist, that introduce some mental overhead. :)
